spec/predicates/cyclonedx.md

```diff
@@ -0,0 +1,71 @@
+# Predicate type: CycloneDX
+
+Type URI: (tentative) https://cyclonedx.org/bom
+
+Version: 1.0.0
+
+## Purpose
+
+A Software Bill of Materials type following the [CycloneDX standard].
+
+This allows representing "exportable", or "published" software artifacts,
+services, vulnerability information, and more. For a complete list of
+capabilities see [CycloneDX Capabilities].
+
+## Prerequisites
+The in-toto [attestation] framework.
+
+## Model
+This is a predicate type that fits within the larger [Attestation] framework.
+
+## Schema
+The schema of this predicate type is documented in the
+[CycloneDX Specification].
+
+### Parsing Rules
+The parsing rules for this predicate type are documented in the
+[CycloneDX Specification].
+
+### Fields
+The fields that make up this predicate type are documented in the
+[CycloneDX Specification].
```

Comment:
I think it would be very helpful for implementers to not have to rely only on the CycloneDX specification document. Could you please provide a list of minimum required or expected fields, and a short description of their purpose? You may then still point to the overall CycloneDX spec for more details.